modules/userId/index.js

```diff
@@ -820,6 +820,13 @@ export function init(config) {
       auctionDelay = utils.isNumber(userSync.auctionDelay) ? userSync.auctionDelay : NO_AUCTION_DELAY;
       updateSubmodules();
     }
+    if (userSync && userSync.userIds && userSync.ppid && getUserIds().hasOwnProperty(userSync.ppid) && typeof getUserIds()[userSync.ppid] === 'string') {
```

Comment:
this won't work with every id provider, including id5, since the result of the `getUserIds()['id5id']` would be an object not a string.

the `eids` array is more standard so might be a better choice?

Reply:
pbjs.getUserIdsAsEids().find(x => x.source === 'pubcid.org').uids[0].id
"876b1f92-a6f9-4639-b939-Ab64eaa9d9"
pbjs.getUserIdsAsEids().find(x => x.source === 'id5-sync.com').uids[0].id
"ID5-ZHMORb_9Ukvpgr1efasddgasdaXF8BU54y7aiyOVKwU0jhu2w"   

seems to work on this obj yeah

Reply:
yea i think that's the better approach to be scalable across how any id provider wants to store the userid object